_test/test_sqw/test_pagedPixelData.m

```diff
@@ -0,0 +1,95 @@
+classdef test_pagedPixelData < TestCase
```

Comment:
I think these tests belong in `test_sqw_file/`. The intention of this PR's tests is to verify that when we call `save(sqw_obj, ...)`, if `sqw_obj` contains paged pixels, that we save all those pixels. We're testing `save` performs the correct actions on the `PixelData` object, not testing the `PixelData` functions.

I would've said the most sensible place for the test you've written would be in `test_sqw_file_read_write.`, but unfortunately that's an old-style test (non-class based) and I'd be hesitant to add to that. My recommendation would be to create a new test class (something like `test_read_then_write`) in `test_sqw_file/` and put your new tests in there.

Reply:
ok will do